frontend/src/app/general/logo/logo.component.html

```diff
@@ -0,0 +1,26 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="flex select-none items-center gap-2">
+  @if (navBarService.logoUrl$ | async) {
+    <img
+      [src]="navBarService.logoUrl$ | async"
+      aria-hidden="true"
+      class="max-h-8 max-w-24"
+    />
+  }
+  <div class="text-2xl text-primary">Capella</div>
```

Comment:
```suggestion
    <div class="text-sm text-primary font-bold text-center">Capella<br />Collaboration</div>
```

Capella might indicate that we're actually looking at Capella, but it's just the wrapper around. Also from a trademarking perspective, I'd make it clearer, e.g., by using the Capella Collaboration term.